gramex/apps/ide/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can we remove whitespace-only changes?

Reply:
Done. My bad, should have sanitized beforehand.